ci-operator/config/konveyor/mig-operator/konveyor-mig-operator-master.yaml

```diff
@@ -0,0 +1,56 @@
+base_images:
+  base:
+    name: base
+    namespace: ocp
+    tag: latest
+  ansible-operator:
+    name: anisble-operator
+    namespace: ocp
+    tag: mig-operator
```

Comment:
I think that you want this tagged as ansible operator and mig-operator to be the image we build

Reply:
+1